sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/resource/AzureStorageUtils.java

```diff
@@ -11,6 +11,11 @@
  * Azure storage resource utility class
  */
 final class AzureStorageUtils {
+
+    private AzureStorageUtils() {
+        throw new IllegalStateException("Utility class");
```

Comment:
no need for this exception?

Reply:
This is from the sonar suggestion.

Reply:
refer: https://sonarcloud.io/project/issues?directories=sdk%2Fspring%2Fspring-cloud-azure-autoconfigure%2Fsrc%2Fmain%2Fjava%2Fcom%2Fazure%2Fspring%2Fcloud%2Fautoconfigure%2Faad%2Fimplementation%2Fjackson&resolved=false&types=CODE_SMELL&id=stliu_azure-sdk-for-java&open=AXtAG3K-00FAjxsg1BbB

Reply:
let's not follow this suggestion

Reply:
Ok, will not follow this.